src/verifier.js

```diff
@@ -1,6 +1,6 @@
 import { STEPS, SUB_STEPS, VERIFICATION_STATUSES } from './constants';
 import debug from 'debug';
-import CERTIFICATE_VERSIONS from './constants/certificateVersions';
+import CERTIFICATE_VERSIONS, { isV3 } from './constants/certificateVersions';
```

Comment:
To be consistent, there's some `this.version === CERTIFICATE_VERSIONS.V1_1` calls in this file too, should you add a `isV1` `isV2` to this as well? Otherwise I'd say go back to `this.version === CERTIFICATE_VERSIONS.V3_ALPHA` below.

Reply:
It is a good point, but we should probably do it across the app. I'll make a note to do the full refactor in the coming days.

Reply:
👍